src/api/entities/CheckpointSchedule/types.ts

```diff
@@ -1,16 +1,6 @@
 import BigNumber from 'bignumber.js';
 
-import { CalendarPeriod } from '~/types';
-
-export type ScheduleDetails = {
+export interface ScheduleWithDetails {
```

Comment:
I never said to rename this one. This should still be called `ScheduleDetails`. The other one was the one you had to rename